packages/fields/src/types/DateTime/Implementation.js

```diff
@@ -68,6 +68,9 @@ class _DateTime extends Implementation {
       }),
     };
   }
+  getBackingTypes() {
+    return { [this.path]: { optional: true, type: 'string | null' } };
```

Comment:
I don't think this is right, isn't DateTime spread across two fields in the db?

Reply:
Yeah, but that is translated at the adapter-hook layer. From the developers point of view they see just the path as a string (at least I'm 90% sure of this, will need to verify this behaviour with tests at some point real soon now).